src/components/AnonymousUsername/index.js

```diff
@@ -12,7 +12,7 @@ const styles = StyleSheet.create({
     fontWeight: 'bold',
     fontSize: 14,
     lineHeight: 16.94,
-    color: COLORS.black,
+    color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from black to white. Please ensure that this change doesn't affect readability on the background it is placed on. If the background is light, this could lead to poor visibility.
```diff
-    color: COLORS.black,
+    color: COLORS.white2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->